lib/absinthe/schema/notation.ex

```diff
@@ -112,12 +164,14 @@ defmodule Absinthe.Schema.Notation do
         {:ok, topic: args.user_id}
       end
 
+      # Both mutations will trigger the subscription
       trigger :gps_event, topic: fn event ->
         event.user_id
       end
 
-      trigger :user_checkin, topic: fn user ->
-        [user.id, user.parent_id]
+      trigger :user_event, topic: fn user ->
```

Comment:
I don't see `user_event` referenced anywhere else in this file, did you mean to change this?